awx/main/migrations/0165_node_and_link_state.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.13 on 2022-07-19 13:54
+
+from django.db import migrations, models
+
+
+def forwards(apps, schema_editor):
+    # All existing InstanceLink objects need to be in the state
+    # 'Established', which is the default, so nothing needs to be done
+    # for that.
+
+    Instance = apps.get_model('main', 'Instance')
+    for instance in Instance.objects.all():
+        instance.node_state = 'ready' if not instance.errors else 'unavailable'
+        instance.save(update_fields=['node_state'])
```

Comment:
this will be a no-op in most cases... but the table size is small in the worst of cases, so I'm fine with all of this.